#Goal
int32 count_goal
---
#result
int32 final_count
---
#feedback
int32 current_count
